test/integration/cli/actions-check-field.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const assert = require('proclaim');
+const runPa11yCli = require('../helper/pa11y-cli');
+
+// Note: we use the JSON reporter in here to make it easier
+// to inspect the output issues. The regular CLI output is
+// tested in the reporter tests
+describe('CLI action "check-field"', () => {
+	let pa11yResponse;
+
+	describe('when the field exists and is unchecked', () => {
+
+		before(async () => {
+			pa11yResponse = await runPa11yCli(`${global.mockWebsiteAddress}/actions-check-field`, {
+				arguments: [
+					'--config', './mock/config/actions-check-field-on.json',
+					'--reporter', 'json',
+					'--ignore', 'warning;notice' // This is so we only deal with errors
+				]
+			});
+		});
+
+		it('checks the field before running tests', () => {
+			assert.isArray(pa11yResponse.json);
+			assert.lengthEquals(pa11yResponse.json, 0);
```

Comment:
These very indirect looking success cases are rather confusing on first glance. I wonder if a comment in the file might help make it all a bit less arcane.  E.g. something like "The test file has an a11y error in the markup. When this action is performed the DOM is manupulated by JavaScript to remove the offending element, hence no a11y errors is proof of this successful action"

It might be a bit repetitive, but could alleviate some serious head-scratching when looking at any of these file in isolation.